src/style.css

```diff
@@ -1,3 +1,61 @@
 body {
   margin: 0;
+  font-family: 'Roboto', serif;
```

Comment:
The font-family is set to 'Roboto', serif. Since 'Roboto' is a sans-serif font, it would be more appropriate to use 'sans-serif' as the fallback font instead of 'serif'.